src/routes/solid-router/reference/data-apis/cache.mdx

```diff
@@ -24,7 +20,35 @@ This cache accomplishes the following:
    Any user based navigation or link click bypasses it.
    Upon revalidation or new fetch the cache is updated.
 
-Using it with a load function:
+### Arguments
+
+| argument | type | description |
+| -------- | ---- | ----------- |
+| fn | `(...args: any) => any` | A function whose return value you'd like to be cached. This function can have any arguments you'd like, and the arguments will be deterministically serialized and used as part of the unique key for the cache. This serialization process uses `JSON.stringify`, but first sorts the key/value pairs of objects to ensure `{ name: 'Ryan', awesome: true }` serializes to the same value as `{ awesome: true, name: 'Ryan' }`. |
```

Comment:
Too much information for the table - can you please pull it out and put it into the beginning section. They should be ~ one sentence (two as an absolute maximum) in the table.